### PR TITLE
Add Marathon service note re: delayed service deployment

### PR DESCRIPTION
**Pull request by SarahStegall**
## Jira Ticket
https://jira.mesosphere.com/browse/DCOS-54051

## Description of changes being made
Sometimes an operator tries launching an application and runs into a resource limit. Then Marathon would back-off and stop trying to launch the application.
With the feature the operator can re-set the back-off and retry launch the delayed application.

## Checklist
Changes made to 1.13 and 1.14 pages:

deploying-services/deployments
gui/services
monitoring/debugging/gui-debugging
monitoring/debugging/stuck-deployment

Also added images:
GUI-Services-Debug
GUI-Services-Enterprise
GUI-Services-Failure-To-Launch

### CODE REVIEW

**Comment by SarahStegall** [comment]
All changes tested and passed in local build.

File: pages/1.13/gui/secrets/index.md [hunk 1]
```diff
@@ -10,6 +10,67 @@ model: /1.13/data.yml
...
 Figure 1 - Secrets page
+
+
+For complete details on creating and managing Secrets, see the [Secrets](/1.13/security/ent/secrets) documentation.
+
+<p class="message--important"><strong>IMPORTANT: </strong>The maximum file size for a secret is approximately 1 MB, subtracting approximately 1 KB for the secret store metadata.</p>
+
+# Creating key-value pair secrets 
+
+1. Log in to the DC/OS UI as a user with the `dcos:superuser` permission.
```
Comment (nLight) [inline pages/1.13/gui/secrets/index.md]: `dcos:superuser` is just as an example here, right? Any user with `dcos:secrets:default:* create` can create a secret.

@DominikDary can probably provide a better insight, but in general it is `dcos:secrets:default:[marathon-service-or-group-path] (create|read|update|delete)`

File: pages/1.13/gui/secrets/index.md [hunk 2]
```diff
@@ -10,6 +10,67 @@ model: /1.13/data.yml
...
+
+    ![New Secret](/1.13/img/new-secret.png)
+
+    Figure 2. New Secret icon
+
+1. In the **ID** box, provide the name of your secret and its path, if any.
+
+1. Type or paste the secret into the **Value** box.
+
+    ![Secret ID/Value Fields](/1.13/img/create-secret.png)
```
Comment (nLight) [inline pages/1.13/gui/secrets/index.md]: Would you mind updating the screenshot please?

Screenshot 2019-08-06 at 12 56 34

File: pages/1.14/gui/secrets/index.md [hunk 3]
```diff
@@ -10,6 +10,68 @@ model: /1.14/data.yml
...
+
+    ![New Secret](/1.14/img/new-secret.png)
+
+    Figure 2. New Secret icon
+
+1. In the **ID** box, provide the name of your secret and its path, if any.
+
+1. Type or paste the secret into the **Value** box.
+
+    ![Secret ID/Value Fields](/1.14/img/create-secret.png)
```
Comment (nLight) [inline pages/1.14/gui/secrets/index.md]: Would you mind updating the screenshot please?

Screenshot 2019-08-06 at 12 56 34

File: pages/1.13/gui/secrets/index.md [hunk 4]
```diff
@@ -10,6 +10,67 @@ model: /1.13/data.yml
...
+
+# Creating secrets from a file 
+
+This procedure describes how to use a file to create a secret using the DC/OS web interface.
+
+1. Log in to the DC/OS UI as a user with the `dcos:superuser` permission.
+1. Click the **Secrets** tab on the left hand navigation menu.
+1. Click the **+** icon in the top right.
+
+    ![New Secret](/1.13/img/new-secret.png)
```
Comment (nLight) [inline pages/1.13/gui/secrets/index.md]: Would you mind updating the screenshot please?

Screenshot 2019-08-06 at 12 59 51

File: pages/1.13/gui/secrets/index.md [hunk 5]
```diff
@@ -10,6 +10,67 @@ model: /1.13/data.yml
...
+
+# Creating key-value pair secrets 
+
+1. Log in to the DC/OS UI as a user with the `dcos:superuser` permission.
+
+1. Open the **Secrets** tab.
+
+1. Click the **+** icon in the top right.
+
+    ![New Secret](/1.13/img/new-secret.png)
```
Comment (nLight) [inline pages/1.13/gui/secrets/index.md]: Would you mind updating the screenshot please?

Screenshot 2019-08-06 at 13 00 58

File: pages/1.13/gui/secrets/index.md [hunk 6]
```diff
@@ -10,6 +10,67 @@ model: /1.13/data.yml
...
+
+1. Type or paste the secret into the **Value** box.
+
+    ![Secret ID/Value Fields](/1.13/img/create-secret.png)
+
+    Figure 3. Creating a new Secret
+
+1. Click **Create**.
+
+# Creating secrets from a file
```
Comment (nLight) [inline pages/1.13/gui/secrets/index.md]: File based secrets are not available in 1.13 https://soak113s.testing.mesosphe.re/#/secrets
I suspect I might have caused this confusion, sorry. We need to remove FBS section from the GUI documentation in 1.13

File: pages/1.13/security/ent/secrets/create-secrets/index.md [hunk 7]
```diff
@@ -97,4 +106,38 @@ The contents of the file (referred to below as `my-secret.txt`) can be any text
   dcos security secrets create -f my-secret.txt developer/my-secret
   ```
 
-  <p class="message--important"><strong>IMPORTANT: </strong>The maximum file size for a secret is approximately one MB, subtracting approximately one KB for the secret store metadata.
+  <p class="message--important"><strong>IMPORTANT: </strong>The maximum file size for a secret is approximately one MB, subtracting approximately one KB for the secret store metadata.</p>
+
+# Creating secrets from a file via the DC/OS UI
```
Comment (nLight) [inline pages/1.13/security/ent/secrets/create-secrets/index.md]: No FBS in DC/OS UI 1.13

File: pages/1.13/security/ent/secrets/create-secrets/index.md [hunk 8]
```diff
@@ -46,13 +47,21 @@ Secrets should include paths, unless you want to allow all services to access it
 
 1. In the **ID** box, provide the name of your secret and its path, if any.
 
+    ![Secret ID Keypair](/1.13/img/GUI-Secrets-Create-New-Keypair.png)
```
Comment (nLight) [inline pages/1.13/security/ent/secrets/create-secrets/index.md]: Please use the create modal window screenshot I provided

**Comment by nLight** [comment]
Hey Sarah! Thanks for the great work! I left some comments mostly around 1.13, 1.14 looks awesome!

File: pages/1.13/gui/secrets/index.md (hunk 1 above)
Comment (SarahStegall) [reply pages/1.13/gui/secrets/index.md]: We'll have to fix this later.

File: pages/1.13/gui/secrets/index.md (hunk 2 above)
Comment (SarahStegall) [reply pages/1.13/gui/secrets/index.md]: This is the old version of this modal dialogue. I have updated the image and it is listed here in this PR.

File: pages/1.14/gui/secrets/index.md (hunk 3 above)
Comment (SarahStegall) [reply pages/1.14/gui/secrets/index.md]: Already done. It's in this PR, it's just not rendering. :(

File: pages/1.13/security/ent/secrets/create-secrets/index.md (hunk 7 above)
Comment (SarahStegall) [reply pages/1.13/security/ent/secrets/create-secrets/index.md]: Noted

File: pages/1.13/gui/secrets/index.md (hunk 6 above)
Comment (SarahStegall) [reply pages/1.13/gui/secrets/index.md]: Noted.

**Comment by SarahStegall** [comment]
All changes to the following pages for both 1.13 and 1.14 have been tested in local build. All built correctly.
deploying-services/deployments
gui/services
monitoring/debugging/gui-debugging
monitoring/debugging/stuck-deployment

**Comment by ck4adventure** [comment]
Please see this PR https://github.com/mesosphere/dcos-docs-site/pull/2532 for migrating the changes onto the new file structure

**Pull request closed without merging**